src/python/ensembl/io/genomio/fasta/process.py

```diff
@@ -23,8 +23,13 @@
 from Bio import SeqIO
 
 from ensembl.io.genomio.utils.archive_utils import open_gz_file
+from ensembl.io.genomio.utils.logging import setup_logging
 from ensembl.utils.argparse import ArgumentParser
 
+# logging references module, but can be overridden by a specific logger
+import logging
+logger = logging # by default
```

Comment:
I was not happy with the `logger` approach either, so that is why I have opted for accessing the `logging.root` instead in my PR.

Reply:
I like the reassigning of `logging`  way,
because in this case you're allowing the module to be used/imported from the outside without calling main function and setup we have in it.
So if you use main -- you define your own logger (however prettified) and assign it to global `logging` (replacing it, basically).
But if you import this module from outside, you preserve logging default settings (perhaps, already modified by the calling/upstream code).

In this case this module itself doesn't import other models with logging.
I think, it's worth to update the root logger to propagate changes downstream, instead as in your PR !207.